script.js

```diff
@@ -1,35 +1,255 @@
+  languages: ['Html/CSS', 'Ruby on rails', 'JavaScrip'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+
+{
+  picture: 'img/HQproject6.svg',
+  projectName: 'Project 6',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languages: ['Html', 'Ruby on rails', 'JavaScrip', 'CSS'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+];
+
+const button2 = document.querySelectorAll('.STP-button');
+const popupContainerDesk = document.querySelector('.popupContainer2');
+
+button2.forEach((i, indice2) => {
+  i.addEventListener('click', () => {
+    const damian2 = `<div class="superPopUpCont">
+    <div class="popupTextBox">
+          <div class="popboxTitle">
+              <h3 id="crossTypo2" class="popProjectTitle"> X </h3>
+              <h3 class="popProjectTitle"> ${popupProjectsDesk[indice2].projectName} </h3>
+          </div>
+          <div class="popCodingLangButtons">
+              <ul class="lists">
+                  <li class="popupLang">${popupProjectsDesk[indice2].languages[0]}</li>
+                  <li class="popupLang">${popupProjectsDesk[indice2].languages[1]}</li>
+                  <li class="popupLang">${popupProjectsDesk[indice2].languages[2]}</li>
+              </ul>
+          </div>
+      </div>
+  
+      <div class="popPicture">
+          <img class="popPicturesPreview" src="${popupProjectsDesk[indice2].picture}" alt="Project Number 1">
+      </div>
+  
+      <div id="popupPdiv">
+          <p class="popupP">
+          ${popupProjectsDesk[indice2].description}
+          </p>
+  
+          <div class="inlineBlockButton">
+          <a href="${popupProjectsDesk[indice2].seeLink}">
+              <button type="button" id="popUnionButton" class="popupButtons">
+                  <h4 class="popButText">See live</h4>
+              </button>
+              </a>
+
+              <a href="${popupProjectsDesk[indice2].sourceLink}">
+              <button type="button" id="popGitHubButton" class="popupButtons">
+                  <h4 class="popButText"> See source</h4>
+              </button>
+              </a>
+          </div>
+      </div>
+      </div>`;
+
+    popupContainerDesk.innerHTML = damian2;
+    popupContainerDesk.classList.add('popup');
+
+    const closex2 = document.getElementById('crossTypo2');
```

Comment:
- [ ] Nice implementing the design, however, As requested by the previous reviewer. Your mobile menu page design should look similar to the original design. 💯
 - [ ] Please improve the technologies section layout 👍🏻 
 - [ ]   It would be better if you make your popup windows scrollable to see the entire page, Actually, the `see source` button and what comes after are missing on your pop-up design kindly consider addressing this 💯 👍🏻 
 - [ ] Let's use the same background color on the active, hover or ... as provided and this will help you to fulfill the design requirements.
![image](https://user-images.githubusercontent.com/79058364/226147124-6a175955-d29e-4fc2-ba39-3d30b5fe1b5c.png)

 - [ ] As requested by the previous reviewer, Kindly make sure your design matches the original one. Actually, your nav menu mobile does not have the full height background, and the display layout, width, padding, and margin ... size, and weight is not respected.

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
|![image](https://user-images.githubusercontent.com/79058364/226146884-4d8bb886-d3ef-4d23-bebb-4fa4940218ef.png) | ![image](https://user-images.githubusercontent.com/79058364/226146766-c2050804-5ded-4923-8442-ce1902525a9b.png) |